open_academy/views/open_academy_views.xml

```diff
@@ -1,4 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
 <odoo>
+    <record id="open_academy_dashboard_view" model="ir.ui.view">
+        <field name="name">Dashboard</field>
```

Comment:
Remember to follow these statements:

```
<record id="view_id" model="ir.ui.view"
  <field name="name">view.name</field>
```

Reply:
> Also, your title is not properly formatted in your description

why?, what's wrong?

Reply:
So based on this example the correct format will be:
```
<record id="open_academy_dashboard_view" model="ir.ui.view">
  <field name="name">dashboard.view</field
```
Am I wrong?

Reply:
Dashboard.name